db/migrations/PostgreSQL/deploy/2/001-auto.sql

```diff
@@ -0,0 +1,50 @@
+--
+-- Created by SQL::Translator::Producer::PostgreSQL
+-- Created on Tue Jun 10 12:27:34 2025
+--
+;
+--
+-- Table: access_code
+--
+CREATE TABLE "access_code" (
+  "id" serial NOT NULL,
+  "code" character(8) NOT NULL,
+  "title" character varying(60),
+  "expires_in" integer NOT NULL,
+  "type" integer NOT NULL,
+  "is_reusable" bit DEFAULT '0' NOT NULL,
+  "is_expired" bit DEFAULT '0' NOT NULL,
+  "created_by" character varying(50) NOT NULL,
+  "created_at" timestamptz DEFAULT CURRENT_TIMESTAMP NOT NULL,
```

Comment:
We should make a relation of many to one between this column and `username` from the `users` table.